core/src/test/scala/org/apache/spark/scheduler/CoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -184,3 +191,47 @@ class CoarseGrainedSchedulerBackendSuite extends SparkFunSuite with LocalSparkCo
     )
   }
 }
+
+/** Simple cluster manager that wires up our mock backend for the gpu resource tests. */
+private class CSMockExternalClusterManager extends ExternalClusterManager {
```

Comment:
It looks like the actual tests that were using this are now gone, we should either add the test back or remove this.  Preferably I think we add it back since I don't think anything else is testing the SchedulerBackend changes.

Reply:
Will add back test case later

Reply:
updated